docs/source/user/jit.rst

```diff
@@ -183,6 +183,25 @@ a file-based cache.  This is done by passing ``cache=True``::
    def f(x, y):
        return x + y
 
+
+
+.. note::
+    Caching of compiled functions has several known limitations:
+
+    - The caching of compiled functions is not performed on a
+      function-by-function basis. The cached function is the the main jit
+      function, and all secondary functions (those called by the main
+      function) are incorporated in the cache of the main function.
+    - Cache invalidation fails to recognize changes in functions defined in a
+      different file. This means that when a main jit function calls
+      functions that were imported from a different module, a change in those
+      other modules will not be detected and the cache will not be updated.
+      This carries the risk that "old" function code might be used in the
+      calculations.
+    - Global variables are treated as constants. The cache will remember the value
+      in the global variable used at compilation. On cache load, the cached
+      function will not rebind to the new value of the global variable.
```

Comment:
```suggestion
    - Global variables are treated as constants. The cache will remember the value
      of the global variable at compilation time. On cache load, the cached
      function will not rebind to the new value of the global variable.
```
Suggest this edit.

Reply:
I merged your suggestion. As this text came from the developer docs, I changed it also there and pushed the change in my latest commit. I didn't make any other changes since your review, only backporting this change.